community/gill-jito-airdrop/anchor/README.md

```diff
@@ -0,0 +1,452 @@
+# Solana Merkle Distributor - Modern Implementation
```

Comment:
i see an anchor folder but no anchor program. What's the use of the folder beyond the codama.config.ts?